### PR TITLE
Delete request type client

### PR DESCRIPTION
**Pull request by kevin-wangg**
Screenshot from 2021-04-15 00-50-54

### CODE REVIEW

File: client/src/components/organisms/style/Modal.scss [hunk 1]
```diff
@@ -231,6 +231,7 @@
     font-size: 34px;
     color: black !important;
     opacity: 1;
+    cursor: pointer;
```
Comment (chamod-gamage) [inline client/src/components/organisms/style/Modal.scss:234]: ty for this lol

File: client/src/components/molecules/RequestTypeDropdown.tsx [hunk 2]
```diff
@@ -88,6 +92,7 @@ const RequestTypeDropdown: FunctionComponent<Props> = (props: Props) => {
 
     // when user clicks delete button request type
     const onOpenDeleteRequestType = () => {
+        console.log("Open delete modal")
```
Comment (chamod-gamage) [inline client/src/components/molecules/RequestTypeDropdown.tsx]: stray log

**Review by meganniu — APPROVED** [review]
Tested and works!
thanks again @kevinwang2040  :))))))))

**Review by chamod-gamage — APPROVED** [review]
LGTM! just a stray console log

**Comment by bonnie-chin** [comment]
@kevinwang2040 HI omg sorry we didn't get to this sooner, these are super small styling fixes -- I'd probably give this like a P3 (I can add this into the task list if y'all want @meganniu)

- bolded text inside of the paragraph should be semibold instead of bold
- more padding on the top between the "Delete Type" text and the top edge of the modal (should match what we have on the bottom)

**Pull request merged**